third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1116,6 +1116,28 @@ func TestAccContainerCluster_withNodePoolAutoscaling(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withNodePoolNamePrefix(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withNodePoolNamePrefix(),
+			},
+			{
+				ResourceName:            "google_container_cluster.with_node_pool_name_prefix",
+				ImportStateIdPrefix:     "us-central1-a/",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"node_pool.0.name_prefix"},
```

Comment:
I don't have a concrete enough idea right now, but I wonder if we could make a feature request in the future for a way to specify these made-up ("virtual"?) fields in Schema that would allow the provider to automatically fill them in from config on import.

Reply:
That's a neat idea! I'll bring it up with the enablement team.